client/src/language/EmbeddedLanguageDocsManager.ts

```diff
@@ -56,6 +56,21 @@ export default class EmbeddedLanguageDocsManager {
     })
   }
 
+  async deleteEmbeddedLanguageDocsFolder (): Promise<void> {
+    logger.debug(`Delete embedded language documents folder: ${this.embeddedLanguageDocsFolder}.`)
+    await new Promise<void>((resolve) => {
+      if (this.embeddedLanguageDocsFolder === undefined) {
+        return
```

Comment:
why don't you resolve in this case? We could get deactivate to wait infintely. We can consider the promise fullfilled since there's nothing to delete.

Reply:
You're right. Done.